routes/routes.js

```diff
@@ -0,0 +1,54 @@
+const mysqlPool = require('../data/config');
+
+//All my routes
+const router = app=>{
+    app.get('/', (req, res)=>{
+        res.send('My First Express API')
+    });
+
+    //fetch all users
+    app.get('/users', (req, res)=>{      
+        mysqlPool.query('SELECT * FROM users', (error, result)=>{
+            if(error) throw error;
```

Comment:
this will break your code when mysql raises an error, use Promises for all async calls and *catch all thrown errors*